ouroboros-consensus/test/infra-test/Test/Ouroboros/Consensus/PeerSimulator/ScheduledChainSyncServer.hs

```diff
@@ -0,0 +1,148 @@
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE RecordWildCards     #-}
+
+module Test.Ouroboros.Consensus.PeerSimulator.ScheduledChainSyncServer (
+  RequestNext (..),
+  FindIntersect (..),
+  ChainSyncServerHandlers (..),
+  ScheduledChainSyncServer (..),
+  scheduledChainSyncServer,
```

Comment:
Looks like `scheduledChainSyncServer` is only needed in this module, perhaps it can be dropped from the export list.